app/src/main/java/org/sagebionetworks/research/mpower/studyburst/StudyBurstActivity.kt

```diff
@@ -0,0 +1,96 @@
+package org.sagebionetworks.research.mpower.studyburst
+
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.activity_study_burst.*
+import kotlinx.android.synthetic.main.study_burst_cell.view.*
+import org.slf4j.LoggerFactory
+
+import org.sagebionetworks.research.mpower.R
+import org.sagebionetworks.research.mpower.TaskLauncher
+import javax.inject.Inject
+
+
+class StudyBurstActivity : AppCompatActivity() {
+    private val LOGGER = LoggerFactory.getLogger(StudyBurstActivity::class.java)
+
+    private val studyBurstViewModel: StudyBurstViewModel by lazy {
+        ViewModelProviders.of(this).get(StudyBurstViewModel::class.java)
+    }
+
+    @Inject
+    lateinit var taskLauncher: TaskLauncher
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+        super.onCreate(savedInstanceState)
+        LOGGER.debug("StudyBurstActivity.onCreate()")
+        setContentView(R.layout.activity_study_burst)
+
+        studyBurstViewModel.getMessage().observe(this, Observer {
+            text -> studyBurstMessage.setText(text)
+        })
+        studyBurstViewModel.getTitle().observe(this, Observer {
+            text -> studyBurstTitle.setText(text)
+        })
+        studyBurstViewModel.getExpires().observe(this, Observer {
+            text -> expiresText.setText(getResources().getString(R.string.study_burst_progress_message, text))
+        })
+
+        studyBurstViewModel.getItems().observe(this, Observer {
+            items ->
+                LOGGER.debug("How many items: " + items?.size)
+                var completed = items?.count { it.completed }
+                LOGGER.debug("Number complted: " + completed)
+                studyBurstStatusWheel.setProgress(completed!!)
```

Comment:
You don't want to be an NPE-lover do you?

[From https://kotlinlang.org/docs/reference/null-safety.html#the--operator]
**The !! Operator**

The third option is for NPE-lovers: the not-null assertion operator (!!) converts any value to a non-null type and throws an exception if the value is null. We can write b!!, and this will return a non-null value of b (e.g., a String in our example) or throw an NPE if b is null:

Reply:
Haha!  No, I don't.  Still getting the hang of kotlin.  I will see if I can clean it up.